modules/ROOT/pages/fd-scatter-gather.adoc

```diff
@@ -0,0 +1,259 @@
+<3> The expression that concatenates the query results.
+
+The message that the Transform card receives from the Set Variable card contains these parts:
+
+----
+payload
+  0
+    payload
+      Name
+      ID
+      Email
+  1
+    payload
+      Name
+      ID
+      Email
+attributes
+vars
+  SG_payload
+    Name
+    ID
+    Email
+----
+
+The fields that are in the variable `SG_payload` can be mapped to the fields of a custom data type that uses the CSV format:
+
+.You can map the fields of the variable to the fields of the custom data type.
+image::fd-sg-5-transform.png[Mapping the fields of the variable to the output]
+<1> The location of the variable in the *Input* pane.
+<2> The mappings from the fields in the variable to the fields in the custom data type.
+
+You drag the `Name` field from the input pane to the `Name` field in the output pane. You do the same for the `ID` and `Email` fields.
+
+The result looks like this, with the actual content differing according to the Salesforce organizations that are queried. The first twelve contacts are from organization A, their IDs all beginning with `0036g000009C`. The remaining contacts are from organization B, their IDs beginning with `0028t000001J`:
+
+----
+Name,ID,Email
+Rose Gonzalez,0036g000009C4WwAAK,rose@edge.com
+Sean Forbes,0036g000009C4WxAAK,sean@edge.com
+Lauren Boyle,0036g000009C4X4AAK,lboyle@uog.com
+Babara Levy,0036g000009C4X5AAK,b.levy@expressl.net
+Josh Davis,0036g000009C4X6AAK,j.davis@expressl.net
+Jane Grey,0036g000009C4X7AAK,jane_gray@uoa.edu
+Arthur Song,0036g000009C4X8AAK,asong@uog.com
+Ashley James,0036g000009C4X9AAK,ajames@uog.com
+Tom Ripley,0036g000009C4XAAA0,tripley@uog.com
+Liz D'Cruz,0036g000009C4XBAA0,ldcruz@uog.com
+Edna Frank,0036g000009C4XCAA0,efrank@genepoint.com
+Avi Green,0036g000009C4XDAA0,agreen@uog.com
+Alden Macfarlane,0028t000001J3BAB2,a.macfarlane@trundle.org
+Tiffani Davalos,0028t000001J3FAH1,t.davalos@trundle.org
+Elly Fenley,0028t000001J7JUp9,efenley@tripple.com
+Richie Fore,0028t000001J3HKJ0,rfore@tripple.com
+Ricarda Cieslak,0028t000001J3WLl9,rcieslak@tripple.com
+Rosaria Jorgensen,0028t000001J3BNM0,rosaria.jorgensen@huttrick.com
+Treasa Barbeau,0028t000001J3BNk3,treasa.barbeau@huttirck.com
+Dwayne Sgro,0028t000001J3EPO2,dsgro@sgroflo.com
+Maude Wilber,0028t000001J4UUy4,mwilber@zhipe.com
+Lashell Rouleau,0028t000001J3NIK6,lrouleau@zhipe.com
+Agnus Diehm,0028t000001J3TCX8,agnus.diehm@aigiant.com
+Pamella Fite,0028t000001J1LOQ3,pamella.fite@aigiant.com
+Kennith June,0028t000001J3POI0,kennith.june@aigiant.com
+Keiko Woods,0028t000001J3SIL1,k.woods@fiephdom.org
+Natashia Horvat,0028t000001J3KSV0,n.horvat@fiephdom.org
+Margarita Ptak,0028t000001J3VVe5,ptakm@greelo.com
+Delena Hodgdon,0028t000001J3OSQ2,hodgdond@greelo.com
+Britt Lafferty,0028t000001J6HUT7,britt.lafferty@hoipolloi.com
+Irving Bates,0028t000001J3IUe6,irving.bates@hoipolloi.com
+Merle Cater,0028t000001J3PPl0,merle.cater@hoipolloi.com
```

Comment:
These emails look real. :0
If they are, please be sure to put a mock email. Same with the ID

Reply:
@matibenav , all of the entries where the ID starts with 036g are from the sample data that are in every Salesforce developer account.
All of the other entries I created. I used a random-name generator to come up with names, make up the IDs myself, and made up the company names in all of the email addresses, too.
I have, however, just tested all of the domains that I made up and found 2 that were in fact being used. So, I changed the corresponding email addresses.